### PR TITLE
Update Chinese documentation

### PR DESCRIPTION
**Pull request by kvartet**

### CODE REVIEW

File: docs/zh_CN/Compression/CompressionReference.rst [hunk 1]
```diff
@@ -19,8 +19,8 @@ Compressor
     :members:
 
 
-module 的包装
-^^^^^^^^^^^^^^^^^^^
+Module Wrapper
```
Comment (squirrelsc) [inline docs/zh_CN/Compression/CompressionReference.rst:22]: miss translation.

File: docs/zh_CN/Compression/pruning.rst [hunk 2]
```diff
@@ -23,4 +23,3 @@ NNI 提供了多种非结构化和结构化剪枝算法。
     Pruners <Pruner>
     Dependency Aware Mode <DependencyAware>
     Model Speedup <ModelSpeedup>
```
Comment (squirrelsc) [inline docs/zh_CN/Compression/pruning.rst:25]: They can be translated.

File: examples/nas/oneshot/darts/README_zh_CN.md [hunk 3]
```diff
@@ -1 +1 @@
-[文档](https://nni.readthedocs.io/zh/latest/NAS/DARTS.html)
+[文档](https://nni.readthedocs.io/en/latest/NAS/DARTS.html)
```
Comment (squirrelsc) [inline examples/nas/oneshot/darts/README_zh_CN.md:1]: Fix the error links. They should link to Chinese version

File: examples/nas/legacy/textnas/README_zh_CN.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # TextNAS: A Neural Architecture Search Space tailored for Text Representation
 
-TextNAS 由 MSRA 提出 正式版本。
+TextNAS 由 MSRA 提出 正式版本。 Official Release.
```
Comment (squirrelsc) [inline examples/nas/legacy/textnas/README_zh_CN.md:3]: Translate it.

File: docs/zh_CN/Compression/QuantizationSpeedup.rst [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+为了支持加速混合精度量化，我们将框架划分为两个部分，前端和后端。  
+前端可以是流行的训练框架，如 PyTorch、TensorFlow 等。 后端可以是 
+为不同硬件设计的推理框架，如 TensorRT。 目前，我们支持 PyTorch 作为前端和 
+TensorRT 作为后端。 为了将 PyTorch 模型转换为 TensorRT 引擎，我们利用 onnx 作为中间图 
+表示。 通过这种方式，我们将 PyTorch 模型转换为 onnx 模型，然后 TensorRT 解析 onnx 
+模型生成推理引擎。 
+
+
+量化感知训练结合了 NNI 量化算法 'QAT' 和 NNI 量化加速工具。
+用户应该设置配置，使用 QAT 算法训练量化模型（请参考 `NNI量化算法 <https://nni.readthedocs.io/en/stable/Compression/Quantizer.html>`__）。
```
Comment (squirrelsc) [inline docs/zh_CN/Compression/QuantizationSpeedup.rst:32]: fix the en link

File: docs/zh_CN/Compression/QuantizationSpeedup.rst [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    output, time = engine.inference(data)
+
+
+请注意，NNI还直接支持后训练量化，请参阅完整的示例以获取详细信息。
+
+
+完整的例子请参考 :githublink:`这里 <examples/model_compress/quantization/mixed_precision_speedup_mnist.py>`。
+
+
+关于 'TensorRTModelSpeedUp' 类的更多参数，你可以参考 `Model Compression API Reference <https://nni.readthedocs.io/en/stable/Compression/CompressionReference.html#quantization-speedup>`__ 。
```
Comment (squirrelsc) [inline docs/zh_CN/Compression/QuantizationSpeedup.rst:87]: fix the en link

File: docs/zh_CN/NAS/Overview.rst [hunk 7]
```diff
@@ -1,114 +1,82 @@
-神经网络结构搜索在 NNI 上的应用
+Retiarii 用于神经网络架构搜索
 =======================================
 
+.. Note:: NNI 最新的 NAS 支持都是基于 Retiarii 框架的，仍在使用早期版本 `NNI NAS v1.0 <https://nni.readthedocs.io/en/v2.2/nas.html>`__ 的用户应尽快将工作迁移到 Retiarii 框架。
```
Comment (squirrelsc) [inline docs/zh_CN/NAS/Overview.rst:4]: fix the en link

File: docs/zh_CN/hpo_benchmark.rst [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+   ./runbenchmark_nni.sh [tuner-names]
+
+该脚本运行基准 “nnivalid”，它由一个回归任务、一个二元分类任务和一个多类分类任务组成。 脚本完成后，您可以在文件夹 results_[time]/reports/ 中找到结果摘要。 要在其他预定义基准上运行，请更改 ``runbenchmark_nni.sh`` 中的 ``benchmark`` 变量。 一些基准在 ``/examples/trials/benchmarking/automlbenchmark/nni/benchmarks``\ 中定义，其他基准测试在 ``/examples/trials/benchmarking/automlbenchmark/automlbenchmark/resources/benchmarks/`` 中定义。 大型基准的一个例子是 "nnismall"，它包括8个回归任务、8个二元分类任务和8个多类分类任务。
+
+默认情况下，该脚本在NNI的所有嵌入式 Tuner 上运行该基准测试。 如果在 [tuner-names] 中提供了一个 Tuner 的列表，它只运行列表中的 Tuner。 目前，支持以下 Tuner："TPE", "Random", "Anneal", "Evolution", "SMAC", "GPTuner", "MetisTuner", "DNGOTuner", "Hyperband", "BOHB"。 也可以评估定制 Tuner。 可参考下一章节。 
+
+默认情况下，该脚本针对指定的基准逐一运行指定的 Tuner。 为了在后台同时运行所有的实验，在 ``runbenchmark_nni.sh`` 中设置 "serialize" 标志为 false。 
+
+注意：SMAC Tuner 、DNGO Tuner 和 BOHB Advisor 必须在运行任何实验之前手动安装。 关于安装SMAC和BOHB的更多细节，请参考 `本页面 <https://nni.readthedocs.io/en/stable/Tuner/BuiltinTuner.html?highlight=nni>`_ 。
```
Comment (squirrelsc) [inline docs/zh_CN/hpo_benchmark.rst:41]: fix the en link.

File: docs/zh_CN/hpo_benchmark.rst [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+
+在自定义 Tuner 上运行预定义的基准测试
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+要运行自定义的基准，在 ``./nni/benchmarks`` 文件夹中添加 benchmark_name.yaml 文件，并在 ``runbenchmark_nni.sh`` 中改变 ``benchmark`` 变量。 参见 ``./automlbenchmark/resources/benchmarks/``，了解一些定义自定义基准的例子。
+
+在自定义 Tuner 上运行基准测试
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+要使用自定义 Tuner，首先要确保 Tuner 继承自 ``nni.tuner.Tuner`` 并正确实现所需的API。 关于实现自定义调谐器的更多信息，请参考 `这里 <https://nni.readthedocs.io/en/stable/Tuner/CustomizeTuner.html>`_ 。 接下来，执行以下步骤：
```
Comment (squirrelsc) [inline docs/zh_CN/hpo_benchmark.rst:51]: fix the en link.

File: docs/zh_CN/hpo_benchmark.rst [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+
+
+#. 用 ``nnictl algo register`` 命令安装自定义 Tuner。 详情请查看 `本文档 <https://nni.readthedocs.io/en/stable/Tutorial/Nnictl.html>`_ 。
```
Comment (squirrelsc) [inline docs/zh_CN/hpo_benchmark.rst:54]: fix the en link.

File: examples/nas/legacy/cream/Cream_zh_CN.md [hunk 11]
```diff
@@ -1 +1 @@
-[文档](https://nni.readthedocs.io/zh/latest/NAS/Cream.html)
+[文档](https://nni.readthedocs.io/en/latest/NAS/Cream.html)
```
Comment (squirrelsc) [inline examples/nas/legacy/cream/Cream_zh_CN.md:1]: fix the en link.

File: examples/nas/legacy/pdarts/README_zh_CN.md [hunk 12]
```diff
@@ -1 +1 @@
-[文档](https://nni.readthedocs.io/zh/latest/NAS/PDARTS.html)
+[文档](https://nni.readthedocs.io/en/latest/NAS/PDARTS.html)
```
Comment (squirrelsc) [inline examples/nas/legacy/pdarts/README_zh_CN.md:1]: fix the en link.

File: docs/zh_CN/Compression/CompressionReference.rst (hunk 1 above)
Comment (kvartet) [reply docs/zh_CN/Compression/CompressionReference.rst]: updated

File: docs/zh_CN/Compression/pruning.rst (hunk 2 above)
Comment (kvartet) [reply docs/zh_CN/Compression/pruning.rst]: updated

File: examples/nas/oneshot/darts/README_zh_CN.md (hunk 3 above)
Comment (kvartet) [reply examples/nas/oneshot/darts/README_zh_CN.md]: Thanks for pointing it out. Below English links have been updated to the Chinese version.

File: examples/nas/legacy/textnas/README_zh_CN.md (hunk 4 above)
Comment (kvartet) [reply examples/nas/legacy/textnas/README_zh_CN.md]: I don't know why some translated sentences are repeated twice, sometimes in English, sometimes in Chinese. So I had to manually delete the repeated sentences......

Comment (squirrelsc) [reply examples/nas/legacy/textnas/README_zh_CN.md]: search it in crowdin, but do not to directly update the output. It will be overwritten next time.

Comment (kvartet) [reply examples/nas/legacy/textnas/README_zh_CN.md]: okay~ crowdin will be modified synchronously

**Comment by QuanluZhang** [comment]
@squirrelsc , thanks teacher Song for the detailed review :)

**Review by QuanluZhang — APPROVED** [review]

**Review by scarlett2018 — APPROVED** [review]

**Pull request merged**